plugins/maximize/plugin.js

```diff
@@ -113,24 +120,62 @@
 			var outerScroll;
 
 			// Saved resize handler function.
-			function resizeHandler() {
-				var viewPaneSize = mainWindow.getViewPaneSize();
-				editor.resize( viewPaneSize.width, viewPaneSize.height, null, true );
+			function resizeHandler( height ) {
```

Comment:
I know this function was here before, but `resizeHandler` is not the most meaningful name. I mean **it only tells in what context the function is called** - here it's a resize listener. I know this is common convention like `clickHandler`, `focusHandler`, etc. But it's hard to say what resize it is related to - editor resize, window resize or maybe something else? And what the function does. Apart from that, looking at the code it is also [called like a regular function without the resize context](https://github.com/ckeditor/ckeditor4/pull/4767/files#diff-08c728d4cbb0c41381231354b02cb3511dc880624a3e8a6319e9d19d34760942R245) which makes it even more confusing.

The **main responsibility of a function name is to tell and describe what the function does**. So here it could be something like `setEditorSize` or `resizeEditor`.